site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md

```diff
@@ -7,8 +7,13 @@ description: >
   This guide covers how you can create webhook fleet autoscaler policy.
 ---
 
-The main difference from the Buffer policy is that the logic on how many target replicas you need is delegated to a separate pod.
-This type of Autoscaler would send an HTTP request to the webhook endpoint every sync period (which is currently 30s) with a JSON body, and scale the target fleet based on the data that is returned.
+In some cases, your game servers may need to use custom logic for scaling your fleet that is more complex that what
+can be expressed using the Buffer policy in the fleetautoscaler. This guide shows how you can extend Agones
+with an autoscaler webhook to implement a custom autoscaling policy.
+
+When you use an autoscaler webhook the logic computing the number of target replicas is delegated to a
+separate pod. The fleetautoscaler will send an HTTP request to the webhook autoscaler's `/scale` endpoint every
```

Comment:
replace "seperate pod"  with something like "separate HTTP/s endpoint, such as one provided by a Kuebrnetes Deployment and Service within the same cluster. WDYT?

Reply:
that sounds better, since it doesn't have to be running in a pod.